apps/src/lab2/views/components/predict.module.scss

```diff
@@ -3,8 +3,20 @@
 .multipleChoiceContainer {
   display: flex;
   align-items: start;
+
   input {
     margin: 4px 4px 0 0;
+
+    // When the radio/checkbox is disabled and not checked, dim option.
+    &:disabled:not(:checked) {
+      opacity: 0.5;
+      cursor: not-allowed;
+
+      + .multipleChoiceLetter,
+      + .multipleChoiceLetter + .multipleChoiceLabel {
+        color: $light_gray_500;
```

Comment:
can you use a dsco variable here?

Reply:
Good idea! Here's what it looks like.

![Screenshot 2025-06-25 at 4 51 44 PM](https://github.com/user-attachments/assets/afcf9b57-f1b9-4277-8304-77bbf0c62d48)